lang/ui.en.json

```diff
@@ -395,6 +399,10 @@
     "defaultMessage": "Show data features",
     "description": "Show data features button text"
   },
+  "data-samples-actions-region": {
+    "defaultMessage": "Data and model actions",
+    "description": "Region label for data samples actions"
+  },
```

Comment:
Feedback for text welcome. It might be nice to avoid the word "actions" given our use of the term to represent a machine learning class.

Reply:
I think that is a move in the right direction.